apps/ledger-live-mobile/src/components/DeviceAction/InstallSetOfApps/Restore.tsx

```diff
@@ -1,36 +1,48 @@
 import React from "react";
 import { Button, Flex, Text } from "@ledgerhq/native-ui";
 import { Trans } from "react-i18next";
-import Illustration from "./RestoreIllustration";
+
+import RestoreIllustrations from "./RestoreIllustrations";
 
 type Props = {
   onConfirm: () => void;
   onReject: () => void;
   deviceName?: string;
+  lastSeenDeviceName?: string;
```

Comment:
For the logic about device models, it would be much better to rely on device model ids (which are properly typed and where we can do a simple equality check (`deviceModelId === DeviceModelId.stax` for instance) rather than on device product names where we have to do stuff like "string.indexOf" (which will probably not change)

Reply:
So you will need (on top of deviceName which will be actually displayed to the user), `deviceModelId` and `lastSeenDeviceModelId`